src/orchestrator.py

```
@@ -0,0 +1,247 @@
+                xml = JUnitXml.fromfile(str(report_path))
+                
+                for suite in xml:
+                    combined.add_testsuite(suite)
+                
+                print(f"Merged {report_file}")
+                
+                # Delete individual reports after merge
+                report_path.unlink()
+                print(f"Deleted {report_file}")
+                
+            except Exception as e:
```

Ruff (0.14.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)